htdocs/accountancy/admin/defaultaccounts.php

```diff
@@ -128,6 +128,7 @@
 	}
 
 	$constname = 'ACCOUNTING_ACCOUNT_CUSTOMER_DEPOSIT';
+	$constname = 'ACCOUNTING_ACCOUNT_SUPPLIER_DEPOSIT';
```

Comment:
This erase previous constant ACCOUNTING_ACCOUNT_CUSTOMER_DEPOSIT. Do we forgot it ?

Reply:
Fixed. We need the two constants eventually